src/scss/config/_scale.scss

```diff
@@ -75,7 +75,7 @@ $spacing-sizes: (
 /// @group config-scale
 $theme-sizes: (
   'page-margin': calc(var(--quarter-shim) + 4vw),
-  'border-width': 3px,
+  'border-width': medium,
```

Comment:
@stacyk This results in the updated HTML as expected, but the border itself is not visible.

![Screen Shot 2022-08-22 at 12 28 03 PM](https://user-images.githubusercontent.com/552316/185971652-03265012-ca02-4a5e-88da-672d6a099e31.jpg)

Reply:
@stacyk Looks good!

![Screen Shot 2022-08-24 at 1 28 43 PM](https://user-images.githubusercontent.com/552316/186484731-cd2c2ce0-9eca-40b4-98ac-c7531d54514e.jpg)